doc/development_process/review_process.rst

```diff
@@ -55,8 +59,8 @@ Workflow
 - The project will use the label system to categorize the pull requests.
 - Changes should not be merged before the minimal time has expired.
 
-Categories/Labels
------------------
+Review time categories/labels
```

Comment:
This is already indicated in the top section, it is about time, keep it general, TSC and Security and labels are not related to time.

Reply:
Ok, I will revert. The reason for the change was due to the headings' levels not being to clear, and somebody maybe understanding this to be the general category/label section.